dts/bindings/kscan/ite,it8xxx2-kscan.yaml

```diff
@@ -14,6 +14,12 @@ properties:
     interrupts:
       required: true
 
+    wucctrl-0:
+      type: phandles
+      description: |
+        configure wakeup controller that the interrupt of specific pin
+        can wakeup low power mode EC or not.
```

Comment:
I'm sorry, but I'm having trouble understanding this updated description; could you please try to rephrase?

Reply:
Done